applications/DEMApplication/python_scripts/DEM_analysis_stage.py

```diff
@@ -31,12 +31,14 @@ class DEMAnalysisStage(AnalysisStage):
     def GetParametersFileName(self):
         return "ProjectParametersDEM.json"
 
-    def GetInputParameters(self):
+    def GetInputParameters(self):   # TODO: deprecated
+        print ("WARNING: Calls to this method (GetInputParameters) will become deprecated in the near future.")
```

Comment:
Is it possible to use the Kratos Logger? So we can select the severity.

Reply:
changed to logger.

Reply:
I can't see the logger here, nor in the other prints